dp_mobility_report/model/preprocessing.py

```diff
@@ -30,11 +30,11 @@ def preprocess_tessellation(tessellation: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
     return tessellation[[const.TILE_ID, const.TILE_NAME, const.GEOMETRY]]
 
 
-def _validate_columns(df: pd.DataFrame) -> pd.DataFrame:
+def _validate_columns(df: pd.DataFrame, timestamps) -> pd.DataFrame:
     if const.UID not in df.columns:
         raise ValueError("Column 'uid' must be present in data.")
     df[const.UID] = df[const.UID].astype(str)
-    if const.TID not in df.columns:
+    if const.TID not in df.columns and timestamps:
```

Comment:
again - why is there no trip ID present if there are no timestamps?

Reply:
trip id is there. Removed this condition